app/models/user.rb

```diff
@@ -25,6 +25,8 @@ class User < ActiveRecord::Base
             format: { with: EMAIL_REGEX },
             if: :email
 
+  scope :pending_cards,    -> { where('cards.review_date <= ?', Time.zone.now) }
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.